packages/flutter/lib/src/rendering/mouse_tracking.dart

```diff
@@ -170,7 +169,7 @@ class _MouseState {
 
   @override
   String toString() {
-    String describeEvent(PointerEvent event) {
+    String describeEvent(PointerEvent? event) {
       return event == null ? 'null' : describeIdentity(event);
     }
     final String describeLatestEvent = 'latestEvent: ${describeEvent(latestEvent)}';
```

Comment:
we can remove describeEvent, and replace this line with:
```suggestion
    final String describeLatestEvent = 'latestEvent: ${describeIdentity(latestEvent)}';
```

Reply:
Done